### ISSUE TITLE
PLEASE respect ports given in the URL to access NRDP via send_nrdp.php!

### ISSUE THREAD

**Issue by ericloyd** [NagiosEnterprises/nrdp#9]
PLEASE respect ports given in the URL to access NRDP via send_nrdp.php!  A quick patch to v1.8 would be as follows, but apparently I can't branch this out.

```
301a302
>     if ($url_parts['port'] != "") { $port=":" . $url_parts['port']; } else { $port="80:"; }
318c319
<             $page = $url_parts['scheme'] . '://' . $url_parts['host'] . $url_parts['path'];

---
>             $page = $url_parts['scheme'] . '://' . $url_parts['host'] . $port . $url_parts['path'];
368c369
<         $fp = fsockopen($url_parts['host'], 80, $errno, $errstr, 30);

---
>         $fp = fsockopen($url_parts['host'], $port, $errno, $errstr, 30);
```

**Comment by ericloyd** [comment]
N/m about the branching.  Duh, was thinking I was able to create branches in the nagios repo. :-)

**Comment by jomann09** [comment]
Thanks Eric, we will get this into NRDP. I'm planning on doing a cleanup sometime this week/weekend on this repository to get it in line with what is currently in XI. There's some differences that aren't merged.

**Comment by ericloyd** [comment]
I branched it out a while ago. There's a pull request out.  Feel free to merge it.

**Comment by jomann09** [comment]
Merged in #10
